api/src/main/java/com/codeforcommunity/dto/team/AddGoalRequest.java

```diff
@@ -0,0 +1,74 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.security.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddGoalRequest extends ApiDto {
+  private Integer goal;
+  private Integer teamId;
```

Comment:
Generally, request DTOs should only contain the info that is in the JSON body. In this case, the API specification has teamId in the path as a path parameter. To support that, you should take this out of the DTO and add another argument to the addGoal interface method.